### PR TITLE
langref table of contents in a separate column on large displays

### LINKED ISSUE #3504: Language reference index in a separate column
I think putting the index in a separate column would be an improvement.
If people like it I'll make a pull request after I refine it.
zig-langref

### PR DESCRIPTION
**Pull request by xackus**
Resolves #3504.

### CODE REVIEW

File: doc/langref.html.in [hunk 1]
```diff
@@ -80,11 +81,35 @@
...
       #contents {
         max-width: 60em;
         margin: auto;
       }
 
+      #index {
+        padding: 0 1em;
+      }
+
+      @media screen and (min-width: 1025px) {
```
Comment (andrewrk) [inline doc/langref.html.in]: Shouldn't this be measured in `em` instead of `px`?

**Comment by andrewrk** [comment]
OK I tried this out both on desktop and with a mobile phone and it works great. Happy to merge this. Thanks for patience, it took a long time to get to this.

File: doc/langref.html.in (hunk 1 above)
Comment (xackus) [reply doc/langref.html.in]: `px` is fine, it scales with resolution.

**Comment by andrewrk** [comment]
It's live now, thanks for the contribution!

**Pull request merged**